packages/web-components/examples/all-charts.html

```diff
@@ -178,6 +191,18 @@ <h2>Pie/Donut</h2>
         </ul>
       </div>
     </datacommons-pie>
+    <datacommons-pie
+      header="Environmental Disasters in California"
+      place="geoId/06"
+      variables="Count_WildfireEvent Count_EarthquakeEvent"
+      donut
+    ></datacommons-pie>
+    <datacommons-pie
```

Comment:
I think you wanted to add perCapita to this one?

Reply:
Removed the datacommons-pie example- I ended up not adding perCapita to the pie chart, because there's currently no tooltips and since pie charts are about a single place, the per capita and non per capita visualizations look the same